diesel_cli/src/main.rs

```diff
@@ -236,57 +236,15 @@ fn run_infer_schema(matches: &ArgMatches) -> Result<(), crate::errors::Error> {
     use crate::print_schema::*;
 
     tracing::info!("Infer schema");
-
     let mut conn = InferConnection::from_matches(matches)?;
-    let mut config = Config::read(matches)?.set_filter(matches)?.print_schema;
-
-    if let Some(schema_name) = matches.get_one::<String>("schema") {
```

Comment:
These configuration overwrites are still used in the new code?

Reply:
Yes, those are applied by `set_filter` and `update_config`